cli-output/src/cli_output.rs

```diff
@@ -1025,7 +1025,9 @@ pub struct CliKeyedEpochReward {
 #[serde(rename_all = "camelCase")]
 pub struct CliEpochRewardsMetadata {
     pub epoch: Epoch,
+    // Deprecated
     pub effective_slot: Slot,
+    // Deprecated
     pub block_time: UnixTimestamp,
```

Comment:
Are these not formally marked as `#[deprecated(...)]` because of the backwards compatibility policy on CLI output?

Reply:
Amazingly, deprecated fields have never come up in `solana-cli-output` before. (Or at least, never been marked.) But yes, thanks, that does seem like a better approach, in case something other than `solana-cli` is using the struct.